configs.example/datamodule/configuration/template_configuration.yaml

```
@@ -15,29 +15,386 @@ input_data:
     end_datetime: "2021-09-01T00:00:00"
 
   nwp:
-    nwp_zarr_path: gs://solar-pv-nowcasting-data/NWP/UK_Met_Office/UKV_intermediate_version_7.zarr
-    history_minutes: 60
-    forecast_minutes: 120
-    time_resolution_minutes: 60
-    nwp_channels:
-      - t # live = t2m
-      - dswrf
-      - dlwrf
-      - hcc
-      - mcc
-      - lcc
-      - vis
-      - r # live = r2
-      - prate # live ~= rprate
-      - si10 # 10-metre wind speed | live = unknown
-    nwp_image_size_pixels_height: 24
-    nwp_image_size_pixels_width: 24
+    ukv:
+      nwp_zarr_path: gs://solar-pv-nowcasting-data/NWP/UK_Met_Office/UKV_intermediate_version_7.zarr
+      history_minutes: 60
+      forecast_minutes: 120
+      time_resolution_minutes: 60
+      nwp_channels: # comment out channels as appropriate
+        - t # 2-metre temperature
+        - dswrf # downwards short-wave radiation flux
+        - dlwrf # downwards long-wave radiation flux
+        - hcc # high cloud cover
+        - mcc # medium cloud cover
+        - lcc # low cloud cover
+        - vis # visability
+        - r # relative humidity
+        - prate # precipitation rate
+        - si10 # 10-metre wind speed | live = unknown
+      nwp_image_size_pixels_height: 24
+      nwp_image_size_pixels_width: 24
+      nwp_provider: ukv
+      start_datetime: "2020-01-01T00:00:00"
+      end_datetime: "2021-09-01T00:00:00"
 
   pv:
     pv_files_groups:
       - label: solar_sheffield_passiv
         pv_filename: gs://solar-pv-nowcasting-data/PV/Passive/ocf_formatted/v0/passiv.netcdf
         pv_metadata_filename: gs://solar-pv-nowcasting-data/PV/Passive/ocf_formatted/v0/system_metadata_OCF_ONLY.csv
+        # This is the list of pv_ml_ids to be sliced from the PV site level data
+    pv_ml_ids:
+      [
```

I would suggest removing the MLIds from the example template, as it makes it a very long and large file.
```suggestion
      []
```

---

or leave in 10 examples